test/data_processing/test_nodes.py

```diff
@@ -60,23 +57,62 @@ def test_simple(self):
             np.array([1.632993161855452, 1.632993161855452]) / np.sqrt(3),
         )
 
+    def test_with_error(self):
+        """Compute error propagation. This is quadratic sum divided by samples."""
+        datum = unp.uarray(
+            [[1, 2, 3, 4, 5, 6]],
+            [[0.1, 0.2, 0.3, 0.4, 0.5, 0.6]],
+        )
+
+        node = AverageData(axis=1)
+        processed_data = node(data=datum)
+
+        self.assertAlmostEqual(processed_data[0].nominal_value, 3.5)
+        self.assertAlmostEqual(processed_data[0].std_dev, 0.15898986690282427)
```

Comment:
Perhaps best to show the math that lead to this number to make the test more transparent.

Reply:
Wrote some comment in https://github.com/Qiskit/qiskit-experiments/pull/481/commits/5f49f5f41dfe0f8c10a258292dcf42905df7007e

I did't turn this into math formula since this means we evaluate the code with another (same) code.